### PR TITLE
Adding ssl

### PR DESCRIPTION
**Pull request by kev5**
Added ssl cert. Added a file `deploy.sh` for openshift ssl integration.

### CODE REVIEW

**Comment by SamiSousa** [comment]
Seems like @bruttozz made most if not all these changes to his pull request #41.

**Comment by kev5** [comment]
That's true @SamiSousa. I was getting an error while `make deploy-openshift` for that branch. However, this works fine for me, since I don't have the CA certs required. Anyway, whatever works for everybody.

**Comment by SamiSousa** [comment]
Could you post a gist of the errors you were getting as a comment on #41 ?

File: README.md [hunk 1]
```diff
@@ -16,7 +15,7 @@ You should use this project if you're interfacing a containerized application in
 
 You'll need:
 
-- [`go`](https://golang.org/dl/)
+- [`go`](https://golang.org/dl/) programming language
```
Comment (SamiSousa) [inline README.md]: A bit redundant. What do you think?

File: README.md [hunk 2]
```diff
@@ -61,7 +60,7 @@ $ make deploy-helm
 ### Deploy broker using Openshift
 
 ```console
-$ make push deploy-openshift
+$ make deploy-openshift
```
Comment (SamiSousa) [inline README.md]: This is good. We should review the readme as a separate pull request.

**Comment by SamiSousa** [comment]
Closing since all the changes here are in #41

**Pull request closed without merging**